### ISSUE TITLE
Notice: Undefined offset: 1

### ISSUE THREAD

**Issue by ochorocho** [zorn-v/nextcloud-social-login#103]
Hey, since the last update i sometimes this error message (reported by sentry, did not experience this myself)

social Login version: 1.16.5

Error message:
```
ErrorException: Notice: Undefined offset: 1
#8 /sociallogin/lib/Controller/LoginController.php(160): OCA\SocialLogin\Controller\LoginController::customOidc
#7 /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php(166): OC\AppFramework\Http\Dispatcher::executeController
#6 /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php(99): OC\AppFramework\Http\Dispatcher::dispatch
#5 /var/www/nextcloud/lib/private/AppFramework/App.php(126): OC\AppFramework\App::main
#4 /var/www/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php(47): OC\AppFramework\Routing\RouteActionHandler::__invoke
#3 [internal](0): call_user_func
#2 /var/www/nextcloud/lib/private/Route/Router.php(297): OC\Route\Router::match
#1 /var/www/nextcloud/lib/base.php(975): OC::handleRequest
#0 /var/www/nextcloud/index.php(42): null
```

Social Login is still working for me. But that error seems somehow problematic.

Thanks.

**Comment by zorn-v** [comment]
Fixed in v1.16.6

**Comment by ochorocho** [comment]
Thanks for your fix. Works now.
